### DISCUSSION TITLE
docker-compose-dev.yaml issue

### DISCUSSION THREAD

**Discussion by gaussiangit** [open-webui/open-webui#4849] [General]
I am using docker-compose-dev.yaml from the documentation for the developement. I get following error and frontend just shows the logo. 

>  frontend-1  | Didn't find package micropip-0.6.0-py3-none-any.whl locally, attempting to load from https://cdn.jsdelivr.net/pyodide/v0.26.1/full/
> frontend-1  | Didn't find package packaging-23.2-py3-none-any.whl locally, attempting to load from https://cdn.jsdelivr.net/pyodide/v0.26.1/full/
> o
> 
> backend-1   | USER_AGENT environment variable not set, consider setting it to identify your requests.
> backend-1   | Process SpawnProcess-1:
> backend-1   | Traceback (most recent call last):
> backend-1   |   File "/usr/local/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
> backend-1   |     self.run()
> backend-1   |   File "/usr/local/lib/python3.11/multiprocessing/process.py", line 108, in run
> backend-1   |     self._target(*self._args, **self._kwargs)
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 76, in subprocess_started
> backend-1   |     target(sockets=sockets)
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 61, in run
> backend-1   |     return asyncio.run(self.serve(sockets=sockets))
> backend-1   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
> backend-1   |     return runner.run(main)
> backend-1   |            ^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
> backend-1   |     return self._loop.run_until_complete(task)
> backend-1   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "uvloop/loop.pyx", line 1517, in uvloop.loop.Loop.run_until_complete
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/uvicorn/server.py", line 68, in serve
> backend-1   |     config.load()
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/uvicorn/config.py", line 473, in load
> backend-1   |     self.loaded_app = import_from_string(self.app)
> backend-1   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
> backend-1   |     module = importlib.import_module(module_str)
> backend-1   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
> backend-1   |     return _bootstrap._gcd_import(name[level:], package, level)
> backend-1   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
> backend-1   |   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
> backend-1   |   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
> backend-1   |   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
> backend-1   |   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
> backend-1   |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
> backend-1   |   File "/app/backend/main.py", line 44, in <module>
> backend-1   |     from apps.rag.main import app as rag_app
> backend-1   |   File "/app/backend/apps/rag/main.py", line 223, in <module>
> backend-1   |     update_embedding_model(
> backend-1   |   File "/app/backend/apps/rag/main.py", line 198, in update_embedding_model
> backend-1   |     app.state.sentence_transformer_ef = sentence_transformers.SentenceTransformer(
> backend-1   |                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/sentence_transformers/SentenceTransformer.py", line 287, in __init__
> backend-1   |     modules = self._load_sbert_model(
> backend-1   |               ^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/sentence_transformers/SentenceTransformer.py", line 1487, in _load_sbert_model
> backend-1   |     module = Transformer(model_name_or_path, cache_dir=cache_folder, **kwargs)
> backend-1   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/sentence_transformers/models/Transformer.py", line 54, in __init__
> backend-1   |     self._load_model(model_name_or_path, config, cache_dir, **model_args)
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/sentence_transformers/models/Transformer.py", line 85, in _load_model
> backend-1   |     self.auto_model = AutoModel.from_pretrained(
> backend-1   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/transformers/models/auto/auto_factory.py", line 564, in from_pretrained
> backend-1   |     return model_class.from_pretrained(
> backend-1   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> backend-1   |   File "/usr/local/lib/python3.11/site-packages/transformers/modeling_utils.py", line 3504, in from_pretrained
> backend-1   |     raise EnvironmentError(
> backend-1   | OSError: Error no file named pytorch_model.bin, model.safetensors, tf_model.h5, model.ckpt.index or flax_model.msgpack found in directory /app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/8b3219a92973c328a8e22fadcfa821b5dc75636a.
> backend-1   | /app

**Comment by CallumJHays** [comment]
Same issue as https://github.com/open-webui/open-webui/discussions/2955
